File: week_10/c_sharp_course/week_1/day_4/composition/starter_code/wizard_management_specs/BroomstickTest.cs
using NUnit.Framework;

namespace WizardManagement { 

  [TestFixture]
  public class BroomstickTest {
    Broomstick broom;

    [SetUp]
    public void Init() 
    {
      broom = new Broomstick("Nimbus",10);
    }

    [Test]
    public void HasBrand(){
      Assert.AreEqual("Nimbus", broom.Brand);
    }

    [Test]
    public void HasSpeed(){
      Assert.AreEqual(10, broom.Speed);
    }

    [Test]
    public void CanFly(){
      Assert.AreEqual("mounting broom, running, skipping, flying!", broom.Fly());
    }

  }
}

File: week_06/day_3/composition/Composition_lab_solution/app/src/test/java/example/codeclan/com/composition_lab_solution/BroomstickTest.java
package example.codeclan.com.composition_lab_solution;

/**
 * Created by user on 28/08/2017.
 */

import static org.junit.Assert.*;
import org.junit.*;

import example.codeclan.com.composition_lab_solution.wizard_management.*;


public class BroomstickTest {

    Broomstick broom;

    @Before
    public void before(){
        broom = new Broomstick("Nimbus",10);
    }

    @Test
    public void hasBrand(){
        assertEquals("Nimbus", broom.getBrand());
    }

    @Test
    public void hasSpeed(){
        assertEquals(10, broom.getSpeed());
    }

    @Test
    public void canFly(){
        assertEquals("mounting broom, running, skipping, flying!", broom.fly());
    }

}
